mock/py/mockbuild/mounts.py

```diff
@@ -205,7 +205,7 @@ def __init__(self, rootObj):
                 FileSystemMountPoint(filetype='tmpfs',
                                      device=device,
                                      path=host_path,
-                                     options="rprivate"),
+                                     options="rprivate,mode=0755"),
```

Comment:
Did you meant 0700 ?

Reply:
I didn't want to disallow anyone from stepping into the directory, but not sure.